src/agent/commands/actions.js

```diff
@@ -286,4 +289,38 @@ export const actionsList = [
             return 'Set npc goal: ' + agent.npc.data.curr_goal.name;
         }
     },
+    {
+    name: '!setPreferredLanguage',
```

Comment:
I don't like writing to the settings file at runtime. This should just change the language for the current session